### PR TITLE
text utilities

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following
1. Text Utilities
2. Gray Text
3. Center Text
4. Small Text
t1
t2

### CODE REVIEW

File: TextUtilities/text.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+.center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.gray-text {
+    color: #6b7280;
+}
```
Comment (Prashantrajput11) [inline TextUtilities/text.css]: code looks clean just a minor suggestion, use consistent color naming. Either use hex code or color names or rgba throughout. And use variables to store colors.  Rest looks good to merge

Comment (prakashsakari) [reply TextUtilities/text.css]: thank you for pointing it out, I had missed to use color variable over here,

Comment (prakashsakari) [reply TextUtilities/text.css]: updated, thanks

**Review by rohanmathur91 — APPROVED** [review]
- Nice use of utility functions 
- Looks good to merge

**Pull request merged**